client/my-sites/plans/jetpack-plans/jetpack-complete-page/view-all-products-link/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useTranslate } from 'i18n-calypso';
+
+import './style.scss';
+
+type Props = {
+	siteSlug?: string;
+};
+
+const ViewAllProductsLink: React.FC< Props > = ( { siteSlug } ) => {
+	const translate = useTranslate();
+
+	return (
+		<a
+			className="view-all-products-link"
+			onClick={ () => recordTracksEvent( 'calypso_view_individual_products_link_click' ) }
+			href={ `https://cloud.jetpack.com/pricing/${ siteSlug }` }
```

Comment:
Another minor nitpick.  Should we hardcode cloud.jetpack.com or just redirect to `/pricing/${ siteSlug }`

I had something similar in a PR and it was pointed out to me there is potential for a user to get sent to an environment they weren't expecting if the base URL is hardcoded.

Reply:
Yes, I see your reasoning, and thanks for reviewing! 😄 but remember, this page is the post-connection page, and it is located on the **wordpress.com** domain (https://wordpress/com/jetpack/connect/plans/complete/:site  , to be exact).  So this link purposely needs to direct the user to the Jetpack Cloud pricing page (hostname being: https://cloud.jetpack.com).
So a relative url won't work here in this scenario. It would end up sending the user to https://wordpress.com/pricing/:site, which is a url that does not exist.

Reply:
> So a relative url won't work here in this scenario. It would end up sending the user to https://wordpress.com/pricing/:site, which is a url that does not exist.

Environments are confusing y'all :)